src/controllers/driverController.js

```diff
@@ -0,0 +1,270 @@
+import bcrypt from "bcrypt";
+import "dotenv/config";
+import getPassword from "../services/createPassword.js";
+import sendEmail from "../services/sendEmail.js";
+import Users from "../../sequelize/models/user.js";
+import Profiles from "../../sequelize/models/profile.js";
+import { Sequelize } from "sequelize";
+import jwt from "jsonwebtoken";
+import { development } from "../../sequelize/config/config.js";
+let sequelize = new Sequelize(development);
+let User = Users(sequelize, Sequelize);
+let Profile = Profiles(sequelize, Sequelize);
+
+class UserController {
+  async registerDatabase(req, res) {
+    const driverRegister = await User.sync({ alter: true });
+    const profileRegister = await Profile.sync({ alter: true });
+    res.send("database created");
+  }
+  async login(req, res) {
+    let isPasswordMatch = "";
+    const project = await User.findOne({ where: { email: req.body.email } });
+    if (project) {
+      isPasswordMatch = await bcrypt.compare(
+        req.body.password,
+        project.password
+      );
+    }
+    if (project === null || !isPasswordMatch) {
+      res.status(404).send({ message: "Incorrect email or password" });
+    } else {
+      const token = jwt.sign(
+        { _id: project.email, role: project.role },
+        process.env.SECRET_KEY
+      );
+      res.status(200).json({
+        token,
+        message: "Login successfully",
+        user: {
+          firstName: project.firstName,
+          lastName: project.lastName,
+          email: project.email,
+          role: project.role,
+        },
+      });
+    }
+  }
+
+  async registerDriver(req, res) {
+    const {
+      firstName,
+      lastName,
+      email,
+      phoneNumber,
+      idNumber,
+      driverLicence,
+      dateofBirth,
+      gender,
+      address,
+    } = req.body;
+    const userpassword = getPassword();
+    const password = await bcrypt.hash(userpassword, 12);
+    const dbDriver = await User.findOne({ where: { email } });
```

Comment:
Please make sure you query the db only after making sure the request is correct. i.e. after checking if email, firstname, lastname, etc. and all the required fields are present in the request, then you can query the db.